W/WIGXJPF/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WIGXJPF"
+version = v"1.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://fy.chalmers.se/subatom/wigxjpf/wigxjpf-latest.tar.gz", "90ab9bfd495978ad1fdcbb436e274d6f4586184ae290b99920e5c978d64b3e6a")
```

Comment:
Isn't there a versioned tarball? A tarball called `-latest.tar.gz` sounds like will be overwriten in place in the future, which means the recipe isn't reproducible.

Reply:
```suggestion
    ArchiveSource("http://fy.chalmers.se/subatom/wigxjpf/wigxjpf-$(version_string).tar.gz", "90ab9bfd495978ad1fdcbb436e274d6f4586184ae290b99920e5c978d64b3e6a")
```

Reply:
Yes there is a versioned tarball. it is: http://fy.chalmers.se/subatom/wigxjpf/wigxjpf-1.13.tar.gz